apps/sourcegraph/codeintel-python/app.yaml

```diff
@@ -0,0 +1,62 @@
+{
+    "id": "/sourcegraph/codeintel-python",
+    "cmd": null,
+    "cpus": 1,
+    "mem": 512,
+    "disk": 0,
+    "gpus": 0,
+    "instances": 1,
+    "executor": "",
+    "requirePorts": false,
+    "acceptedResourceRoles": ['*'],
+    "container": {
+        "type": "DOCKER",
+        "docker": {
+            "image": "sourcegraph/codeintel-python",
+            "privileged": false,
+            "forcePullImage": false
+        },
+        "portMappings": [
+            {
+                "containerPort": 2087,
+                "hostPort": 0,
+                "servicePort": 10013,
+                "protocol": "tcp",
+                "name": "main",
+                "labels": {}
+            }
+        ]
+    },
+    "networks": [
+        {
+            "mode": "container/bridge"
+        }
+    ],
+    "healthChecks": [
+        {
+            "protocol": "MESOS_TCP",
+            "port": 2087,
```

Comment:
I changed this to `"portIndex": 0` inside Marathon itself, not sure if that changed anything because after redeploying it seemed to not work, but now the python codeintel stuff is working, so I'm not sure what changed but it might have been this.

This also apparently affects the health check used on marathon-lb, which is how I noticed this was set, so I think you do want this to not be 2087 since that's only the port inside the container, not the port used externally.